### ISSUE TITLE
cannot delete import statements

### ISSUE THREAD

**Issue by just6660** [MajeedKazemi/nova-editor#597]
![image](https://user-images.githubusercontent.com/63805941/171654123-a201d86c-5a37-4f61-958d-6c02febe224e.png)
no way to delete import statements

**Comment by MajeedKazemi** [comment]
Yes, this is a known issue. though you are able to delete it from the
beginning of the line, but it doesn't really make sense and the experience
is not nice.
I think that it should simply delete the whole statement when the user
presses delete or backspace in either of the empty holes.
